### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/deployment/https.md`

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates deployment/https.md in Japanese.

To all reviewers. This section is only separated from #1948 (with few changes), but please review it.

### CODE REVIEW

File: docs/ja/docs/deployment/https.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# HTTPSについて
+
+HTTPSを、「有効」か「無効」にされるだけの何かだと考えることは簡単です。
+
+しかし実態は、より複雑です。
+
+!!! tip "豆知識"
+    もし急いでいたり、気にならない場合は、次のセクションに進んで構築手順を確認して下さい。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/https.md:8]:
```suggestion
    もし急いでいたり、気にならない場合は、次のセクションに進んで別の技術による構築手順を確認して下さい。
```

employ "with different techniques".

File: docs/ja/docs/deployment/https.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    * 期限切れを起こします。
+    * そうなった場合、サードパーティから再取得する必要があります。
+* 接続の暗号化はTCPレベルで行われます。
+    * HTTPよりも1つ下のレイヤーです。
+    * よって、証明と暗号化はHTTPの前に行われます。
+* TCPは「ドメイン」を認識しません。IPアドレスについてのみ認識します。
+    * リクエストされた特定のドメインに関する情報はHTTPデータの中に入ります。
+* HTTPS証明書は特定のドメインであることを「証明」します。しかし、どのドメインで扱われるか知る前にプロトコルと暗号化はTCPレベルで行われます。
+* デフォルトでは、これはIPアドレス毎に一つのHTTPS証明書しか持てないことを意味します。
+    * サーバの大きさやその中にあるそれぞれのアプリケーションの大きさに依りません。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/https.md:26]:
```suggestion
    * サーバーの大きさやその中にあるそれぞれのアプリケーションの大きさに依りません。
```
Consistency of spelling variants.

File: docs/ja/docs/deployment/https.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    * しかし、解決方法はあります。
+* <a href="https://en.wikipedia.org/wiki/Server_Name_Indication" class="external-link" target="_blank"><abbr title="Server Name Indication">SNI</abbr></a>と呼ばれるTLSプロトコル (HTTPの前にTCPレベルで暗号化を行う) への拡張機能が存在します。
+    * このSNI拡張機能により、ひとつのサーバ (一つのIPアドレスをもつ) に複数のHTTPS証明書を持たせることができて、複数のHTTPSドメイン/アプリケーションにサービスを提供できるようになります。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/https.md:29]:
```suggestion
    * このSNI拡張機能により、ひとつのサーバー (一つのIPアドレスを持つ) に複数のHTTPS証明書を持たせることができて、複数のHTTPSドメイン/アプリケーションにサービスを提供できるようになります。
```
Consistency of spelling variants.

File: docs/ja/docs/deployment/https.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    * これを動作させるためには、サーバ上でパブリックIPアドレスを持ってアクセスを待ち受けている一つのコンポーネント (プログラム) にサーバ内の全てのHTTPS証明書をもたせなければなりません。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/https.md:30]:
```suggestion
    * これを動作させるためには、サーバー上でパブリックIPアドレスを持ってアクセスを待ち受けている一つのコンポーネント (プログラム) にサーバー内の全てのHTTPS証明書を持たせなければなりません。
```

**Review by sh0nk — COMMENTED** [review]
Just nitpicks!

**Comment by tokusumi** [comment]
Thank you @sh0nk !
I accepted your suggestions. And fixed some weird translation.
could you check it again?

**Review by sh0nk — APPROVED** [review]
Looks great! Thank you! 👍

**Comment by tiangolo** [comment]
📝 Docs preview for commit 846636fcfad5e666cbac22bbe12ccd57b49b3de7 at: https://649a13ba443f2f103c861546--fastapi.netlify.app

**Comment by tiangolo** [comment]
It seems this was handled in https://github.com/tiangolo/fastapi/pull/10298, so I'm gonna close this one. Thank you! ☕

**Pull request closed without merging**